### PR TITLE
Robotappiumsourcecode

### PR DESCRIPTION
**Pull request by Infobeans-Anujita**
The common.robot file has been modified so that we can parameterised through command line (all parameters).

### CODE REVIEW

File: Tests/Create_survey_demo.robot [hunk 1]
```diff
@@ -28,7 +28,9 @@ Test_CreateSurvey
 Test_ClickCreateSurvey
 
     [Documentation]  click on create survey
-    Given User is on create survey page again
+    Given User is on create survey page
```
Comment (ersuniljoshi) [inline Tests/Create_survey_demo.robot]: Why these changes, please revert back.

File: Tests/Resources/Common.robot [hunk 2]
```diff
@@ -1,14 +1,20 @@
 *** Settings ***
 Library  AppiumLibrary
+Library  String
 
 *** Variables ***
-${REMOTE_URL}  http://localhost:4444/wd/hub
+${hub_url}  http://localhost:4444/wd/hub
+${deviceName}
```
Comment (ersuniljoshi) [inline Tests/Resources/Common.robot]: Default value of device???

File: Tests/Resources/KeywordFileRobotAppium.robot [hunk 3]
```diff
@@ -7,7 +7,6 @@ Resource  ./PO/commenttypequestion.robot
 Resource  ./PO/dropdowntypequestion.robot
 Resource  ./PO/matrixtypequestion.robot
 Resource  ./PO/multipletypequestion.robot
```
Comment (ersuniljoshi) [inline Tests/Resources/KeywordFileRobotAppium.robot]: why we need these changes?

File: Tests/Resources/locator.robot [hunk 4]
```diff
@@ -23,7 +23,7 @@ ${create_survey_button}  xpath=//android.webkit.WebView[@content-desc='Survey Mo
 ${new_survey}  xpath=//android.widget.Button[@text='New Survey']
 ${new_survey_create_input_field}  xpath=//android.widget.EditText[@resource-id='com.surveymonkey:id/input_field']
 ${new_survey_create}  xpath=//android.widget.Button[@text='Create']
-${wrong_create_survey}  xpath=//android.widget.Button1
+${wrong_create_survey}  xpath=//android.widget.Button
```
Comment (ersuniljoshi) [inline Tests/Resources/locator.robot]: why we need this changes?

File: Tests/Create_survey_demo.robot [hunk 5]
```diff
@@ -28,7 +28,9 @@ Test_CreateSurvey
...
+    And User creates new survey
+
 
 Test_CreateSurveyAgain
```
Comment (ersuniljoshi) [inline Tests/Create_survey_demo.robot]: ReadMe Missing :), please add...

Comment (Infobeans-Anujita) [inline Tests/Create_survey_demo.robot]: Done

File: Tests/Resources/Common.robot (hunk 2 above)
Comment (Infobeans-Anujita) [inline Tests/Resources/Common.robot]: Done
